app/webpage/main.js

```diff
@@ -0,0 +1,97 @@
+$(document).ready(function () {
+  //object for easy manipulation of title, description, and day of week of task
+  var task = {
+    title: '',
+    description: '',
+    weekday: 0
+  }
+  
+  //initiation of the webpage's first screen
+  $("#password").hide();
+  $("#sign-up").hide();
+  $("#option-help").hide();
+  $("#week-scheduler").hide();
+  $("#email").val("");
+  $("#pswd").val("");
+  
+/* lgin-user-01, plan-head-01 */
+  $("#log-email").click(eValidate);
+  
+/* lgin-pswd-01 */
+  $("#log-pswd").click(pswdValidate);
+  
+/* lgin-npwd-01 */
+  $("#log-create").click(signupValidate);
+  
+/* plan-head-02-03 */
+  $("#help").click(helpText);
+  
+  $(".wkday").click(function() { setDay($(this).text(), task); });
+});
+
+/* lgin-user-01 */
+function eValidate() {
+  if($("#email").val() == "" || $("#email").val() == undefined){
```

Comment:
Question....
I'm going to have to be making an Ajax call later in that function. Would it still be efficient to just save $("#email") to a variable, rather than $("#email").val()? That way I could call the identity to .hide() later too.
I don't know how much searching .val() might do, but I wanted to double check that it would work.